package.json

```diff
@@ -51,7 +51,7 @@
     "graphology-traversal": "^0.3.1",
     "graphology-types": "^0.24.7",
     "js2xmlparser": "^4.0.2",
-    "jsforce": "2.0.0-beta.22",
+    "jsforce": "2.0.0-beta.23",
```

Comment:
Necessary for the new `encoding` option in jsforce so we can consume the binary response stream for the metadata zip.